### DISCUSSION TITLE
Lightspark Plugin in Tor Browser

### DISCUSSION THREAD

**Discussion by Fapkin69** [lightspark/lightspark#637] [Q&A]
Greeting, I am having an issue using Lightspark in Tor Browser to view flash content. I have the windows.exe 64-bit installed. What is confusing me is this statement, "this installer will include the browser plugins, but they will not be installed automatically." How do I go about installing them to my browser.

**Comment by dbluelle** [comment]
this is already discussed in https://github.com/lightspark/lightspark/discussions/627
